src/org/opensha/sha/earthquake/faultSysSolution/ruptures/strategies/ClusterConnectionStrategy.java

```diff
@@ -304,4 +305,11 @@ public static void main(String[] args) throws IOException {
 		System.out.println("DONE");
 	}
 
+	/**
+	 * @return the clusterConnectionCount
```

Comment:
in line 111 this is called "the number of connections added".

Reply:
done - well spotted